crates/conjure-cp-core/src/ast/expressions.rs

```diff
@@ -1334,6 +1367,17 @@ impl Display for Expression {
 
             Expression::PairwiseSum(_, a, b) => write!(f, "PairwiseSum({a}, {b})"),
             Expression::PairwiseProduct(_, a, b) => write!(f, "PairwiseProduct({a}, {b})"),
+
+            Expression::LexLt(_, a, b) => write!(f, "LexLt({a}, {b}"),
```

Comment:
should be <lex and <=lex assuming Display is supposed to be valid Essence where possible?